test/xebow_test.exs

```diff
@@ -0,0 +1,7 @@
+defmodule XebowTest do
+  use ExUnit.Case
+
+  test "holds the xebow layout" do
```

Comment:
:information_source: (optional)

Suggestion for a subtle change in description: `"has a layout"`. As in `Xebow` is the owner of the layout and provides it to others via `.layout`.